tech.json

```diff
@@ -398,7 +398,8 @@
                   "otherRequires": [],
                   "note" : [
                     "A mid-air morph that has to be done in a 4-tile-high space, into a morph tunnel at the top of the space.",
-                    "It's a lot more precise than with more room. Turn off HiJump before attempting this."
+                    "It's a lot more precise than with more room. Turn off HiJump before attempting this.",
+                    "Most applications of this tech are in places where wall jump instant morphing is not possible."
```

Comment:
Sounds fine, but it should also be included in those locations, because of collectible wall jump.